### PR TITLE
Further scrollbar fixes

### PR DESCRIPTION
**Pull request by peppy**
Just comments

### CODE REVIEW

File: osu.Framework/Graphics/Containers/ScrollContainer.cs [hunk 1]
```diff
@@ -152,6 +152,7 @@ private void updateSize()
         {
             var newAvailableContent = content.DrawSize[scrollDim];
 
+            // ensure we only update scrollbar when something has changed, to avoid transform helpers getting much slower than we want them.
```
Comment (Tom94) [inline osu.Framework/Graphics/Containers/ScrollContainer.cs]: "slower" still sounds like a microoptimization. Can this be more clear about that we do not want to reset tweening?

Comment (peppy) [inline osu.Framework/Graphics/Containers/ScrollContainer.cs]: it's not even a micro-optimisation, as i said (it avoids at least two objects being created every update) but sure

Comment (Tom94) [inline osu.Framework/Graphics/Containers/ScrollContainer.cs]: Aren't there way more objects created every frame anyway as of now? Also, regardless of whether it's "micro" or not, my point was that the _semantic_ difference was unclear and needs to be highlighted.

Comment (peppy) [inline osu.Framework/Graphics/Containers/ScrollContainer.cs]: yep, we have a lot of optimising to be done on per-frame stuff, but that can come much later when we figure out how much overhead GC is consuming (last check it was 20-30%)

**Pull request merged**